client/sites/marketing/tools/index.tsx

```diff
@@ -20,17 +24,46 @@ import './style.scss';
 export default function MarketingTools() {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
-	const recordTracksEvent = ( event: string ) => dispatch( recordTracksEventAction( event ) );
+	const [ searchTerm, setSearchTerm ] = useState( '' );
 	const selectedSiteSlug: T.SiteSlug | null = useSelector( getSelectedSiteSlug );
 	const siteId = useSelector( getSelectedSiteId ) || 0;
 
+	const recordTracksEvent = useCallback(
+		( event: string, properties?: Record< string, string > ) =>
+			dispatch( recordTracksEventAction( event, properties ) ),
+		[ dispatch ]
+	);
+
 	const marketingFeatures = getMarketingFeaturesData(
 		selectedSiteSlug,
 		recordTracksEvent,
 		translate,
 		localizeUrl
 	);
 
+	const marketingFeaturesFiltered = useMemo( () => {
+		return marketingFeatures.filter(
+			( feature ) =>
+				feature.title.toLowerCase().includes( searchTerm.toLowerCase() ) ||
+				feature.description.toLowerCase().includes( searchTerm.toLowerCase() )
```

Comment:
I wonder if this is more readable and performant.

```suggestion
				(feature.title + feature.description).toLowerCase().includes( searchTerm.toLowerCase() )
```

Reply:
I don't think performance is a problem here, if you have strong feelings I can change it

Reply:
Performance isn't. If you don't prefer my version for readability then it's OK.